Sources/MapboxDirectionsCLI/CodingOperation.swift

```diff
@@ -3,7 +3,11 @@ import MapboxDirections
 import Turf
 import CoreLocation
 
-private let BogusCredentials = DirectionsCredentials(accessToken: "pk.feedCafeDadeDeadBeef-BadeBede.FadeCafeDadeDeed-BadeBede")
+let accessToken: String? =
+    ProcessInfo.processInfo.environment["access_token"] ??
```

Comment:
`MAPBOX_ACCESS_TOKEN` is more commonly used by Mapbox tools that get the token from the environment. It’s less likely to collide with a local variable in a shell script or an environment variable expected by another tool.